compiler/aarch64/codegen/GenerateInstructions.hpp

```diff
@@ -578,6 +578,28 @@ TR::Instruction *generateTrg1Src2ExtendedInstruction(
                    uint32_t shiftAmount,
                    TR::Instruction *preced = NULL);
 
+/*
+ * @brief Generates src2-to-trg indexed element instruction
+ * @param[in] cg : CodeGenerator
+ * @param[in] op : instruction opcode
+ * @param[in] node : node
+ * @param[in] treg : target register
+ * @param[in] s1reg : source register 1
+ * @param[in] s2reg : source register 2
+ * @param[in] index : index
```

Comment:
The same comment for the description for `index`.

Reply:
Changed the description to `index of element in s2reg.`